npm-shrinkwrap.json

```diff
@@ -328,9 +328,9 @@
       "resolved": "git+https://github.com/mozilla/eslint-plugin-fxa.git#41504c9dd30e8b52900c15b524946aa0428aef95"
     },
     "fxa-auth-db-mysql": {
-      "version": "1.95.0",
+      "version": "1.95.1",
```

Comment:
@vladikoff made a `1.95.2` tag directly from master yesterday so it looks like you might need to rebase the `train-95` branch to that before opening this PR?

```
* |  (tag: v1.95.2) 5df01c70 vladikoff Release v1.95.2
* |  2e8e6747 Vlad Filippov feat(db): allow BMP chars in device name (#2053) r=rfk,jbuck
| *    57a7f996 Phil Booth Merge branch 'master' into phil/2106
| |\
| |/
|/|
* |    c6486ba0 Ryan Kelly Merge branch 'train-95'
|\ \
| * |  (HEAD -> train-95, tag: v1.95.1, origin/train-95) 187943d8 Ryan Kelly Release v1.95.1
```